include/rmm/device_buffer.hpp

```diff
@@ -392,4 +392,23 @@ class device_buffer {
       mr::get_default_resource()};  ///< The memory resource used to
                                     ///< allocate/deallocate device memory
 };
+
+/**--------------------------------------------------------------------------*
+ * @brief Copies rmm::device_buffer to a preallocated host buffer.
+ *
+ * Copies device memory asynchronously on the specified stream
+ *-------------------------------------------------------------------------**/
+void copy_to_host(const device_buffer& db, void* hb, cudaStream_t stream = 0) {
```

Comment:
Doc params.

Reply:
Also include `@throws` message for the exception cases.

Reply:
Thanks. Added. Also documented exceptions thrown 🙂